### PR TITLE
Add org.gnome.Decibels

### PR DESCRIPTION
**Pull request by vixalien**
### Please confirm your submission meets all the criteria

Decibels is an app for the GNOME Desktop that plays audio files. The app id changed from `com.vixalien.decibels` to `org.gnome.Decibels`. I will mark the former app as "End of life" when this one gets accepted.

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

(cc @cassidyjames)

### CODE REVIEW

File: org.gnome.Decibels.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "blueprint-compiler",
```
Comment (hfiguiere) [inline org.gnome.Decibels.json:35]: this is missing the cleanup rule to remove it completely after build.

File: org.gnome.Decibels.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+          "tag": "v0.12.0"
+        }
+      ]
```
Comment (cassidyjames) [inline org.gnome.Decibels.json:43]:
```suggestion
      ],
      "cleanup": ["*"]
```

File: org.gnome.Decibels.json (hunk 1 above)
Comment (vixalien) [reply org.gnome.Decibels.json:35]: fixed!

**Comment by bbhtt** [comment]
bot, build org.gnome.Decibels

**Comment by cassidyjames** [comment]
I think it's failing now since it's pointing to the 0.1.7 release, which was before the app ID change; we should make a new release on GitLab, then update the manifest to point to it.

**Comment by vixalien** [comment]
bot, build org.gnome.Decibels

**Comment by vixalien** [comment]
works well on my end!!

**Review by cassidyjames — APPROVED** [review]
Huzzah!

**Comment by vixalien** [comment]
Oh, Decibels is in [GNOME Incubator](https://gitlab.gnome.org/GNOME/Incubator/decibels) which means it's not a core app yet, but you it's on track to become one.

One of the things to do while in Incubator is to transition to the org.gnome namespace.

**Comment by sophie-h** [comment]
> I've been given [this list](https://apps.gnome.org/_api/official-gnome-apps.json) by @sophie-h, and Decibels is not there. So I don't think the rename should happen until Decibels is an official GNOME app?

That list is for which apps should be certified as GNOME (official GNOME software.) The list for the use of the `org.gnome` app id prefix is documented here https://docs.flathub.org/docs/for-team-members/review#links

**Comment by vixalien** [comment]
It's verified now! Thanks!!!

**Pull request closed without merging**